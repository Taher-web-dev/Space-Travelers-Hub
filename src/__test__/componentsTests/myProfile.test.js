import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import MyProfile from '../../components/myProfile/myProfile';

describe('MyProfile', () => {
  it('renders correctly', () => {
    const tree = render(<Provider store={store}><MyProfile /></Provider>);
    expect(tree).toMatchSnapshot();
  });
});
