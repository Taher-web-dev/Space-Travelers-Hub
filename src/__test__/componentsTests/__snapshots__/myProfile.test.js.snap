// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyProfile renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        style="width: 90%; margin: 1.25% 5%; display: flex; gap: 30px;"
      >
        <div
          style="width: 100%;"
        >
          <h2>
            My Missions
          </h2>
        </div>
        <div
          style="width: 100%;"
        >
          <h2>
            My Rockets
          </h2>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      style="width: 90%; margin: 1.25% 5%; display: flex; gap: 30px;"
    >
      <div
        style="width: 100%;"
      >
        <h2>
          My Missions
        </h2>
      </div>
      <div
        style="width: 100%;"
      >
        <h2>
          My Rockets
        </h2>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
